core/station/impl/src/services/system.rs

```diff
@@ -386,24 +406,73 @@ impl SystemService {
     pub async fn init_canister(&self, input: SystemInit) -> ServiceResult<()> {
         let mut system_info = SystemInfo::default();
 
-        if input.admins.is_empty() {
-            return Err(SystemError::NoAdminsSpecified)?;
+        if input.users.is_empty() {
+            return Err(SystemError::NoUsersSpecified)?;
         }
 
-        if input.admins.len() > u16::MAX as usize {
-            return Err(SystemError::TooManyAdminsSpecified {
+        if input.users.len() > u16::MAX as usize {
+            return Err(SystemError::TooManyUsersSpecified {
                 max: u16::MAX as usize,
             })?;
         }
 
-        // adds the default admin group
-        init_canister_sync_handlers::add_initial_groups();
+        let admin_quorum = calc_initial_quorum(input.users.len() as u16, input.quorum);
```

Comment:
looks like user quorum

Reply:
Refactored the quorum input so there is required admin_quorum and operator_quorum when default policies are created.